ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -60,12 +60,6 @@ message CreatedEvent {
   // Optional
   Record create_arguments = 4;
 
-  // Opaque representation of contract payload intented for forwarding
```

Comment:
I am proposing to postpone extension of the explicit disclosure to a later time, since that API change would make more sense if the corresponding command.proto changes land together with these.